packages/core/src/extractors/split.ts

```diff
@@ -1,12 +1,12 @@
 import type { Extractor } from '../types'
 import { isValidSelector } from '../utils'
 
-export const splitCode = (code: string) => code.split(/\\?[\s'"`;={}]+/g).filter(isValidSelector)
+export const splitCode = (code: string) => [...new Set(code.split(/\\?[\s'"`;={}]+/g))].filter(isValidSelector)
```

Comment:
I think it does the same job as his name, just to simply split the code. Deduplication should be handed over to the next step. (Just my opinion 😄)